src/chain/node.js

```diff
@@ -141,18 +144,24 @@ async function topBlock () {
   return this.api.getTopHeader()
 }
 
-async function poll (th, { blocks = 10, interval = 500, allowUnsynced = false } = {}) {
```

Comment:
`allowUnsynced` was implemented in https://github.com/aeternity/aepp-sdk-js/pull/954 and used only in FireEditor (added in https://github.com/aeternity/fire-editor/commit/85ed8a39be2024b28eb873d7261ae53d90f95a74)

I can't find any explanation of reasoning in both repositories, but as I got the `allowUnsynced` flag makes SDK check `txInfo.callInfo.height`, but from the first view it should be the same as `tx.blockHeight`. It was made to improve performance, but I don't think that it has significant benefits in comparison to using calldata library and other improvements that we introduced recently. SDK tests work relatively fast without `allowUnsynced`, switching this flag to true doesn't improve performance. So I'm proposing to drop it. cc: @nikita-fuchs

Reply:
sounds good to me